readme.md

```diff
@@ -262,15 +262,14 @@ See [Lighthouse Architecture](./docs/architecture.md).
 
 Yes! Details in [Lighthouse configuration](./docs/configuration.md).
 
-### What is "Do Better Web"?
```

Comment:
fyi @ebidel :)

Reply:
Aye. I figured this was mostly not necessary to call out these days, but I'm open to whatever.